http-client/src/test/groovy/io/micronaut/http/client/ClientIntroductionAdviceSpec.groovy

```diff
@@ -194,6 +249,21 @@ class ClientIntroductionAdviceSpec extends Specification {
     static interface MyClient extends MyApi {
     }
 
+    @Client('/accept')
+    @Requires(property = 'spec.name', value = 'ClientIntroductionAdviceSpec')
+    static interface AcceptTypeClient {
+        @Get
+        String index()
+
+        @Get(value = "/xml", consumes = MediaType.APPLICATION_XML)
+        String xml()
+
+        @Consumes(MediaType.APPLICATION_XML)
+        @Get("/xml-annotated")
```

Comment:
```suggestion
        @Get("/xml")
```

Reply:
Done in [6c3106b](https://github.com/micronaut-projects/micronaut-core/pull/10685/commits/6c3106b2a3d55b0587eb04188547d10d5980fe63)